app/src/main/res/layout/feeditem_fragment.xml

```diff
@@ -101,6 +101,17 @@
             android:layout_marginTop="8dp"
             android:visibility="gone" />
 
+
+        <TextView
+            android:id="@+id/noMediaMsg"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="8dp"
+            android:visibility="gone"
+            android:textAlignment="center"
+            android:textStyle="italic"
+            android:text="Item is read-only"/>
```

Comment:
this string needs to be localized in strings.xml so it can be translated?  Also maybe it should read `No audio/videos files`?

Reply:
@keunes do you have choice of wording?